app/controllers/members_controller.rb

```diff
@@ -49,6 +49,27 @@ def view_followers
     @followers = @member.followers.paginate(:page => params[:page])
   end
 
+  EMAIL_TYPE_STRING = {
+    send_notification_email: "Inbox Notification",
+    send_planting_reminder: "Planting Reminder"
+  }
+
+  def unsubscribe
+    begin
+      verifier = ActiveSupport::MessageVerifier.new(ENV['RAILS_SECRET_TOKEN'])
+      decrypted_message = verifier.verify(params[:message])
+
+      @member = Member.find(decrypted_message[:member_id])
+      @type = decrypted_message[:type]
+      @member.update_attributes(@type => false)
+
```

Comment:
good suggestion! Just learned <code>update_attribetes</code> is an alias of <code>update</code>!